bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/ResourceLimiter.java

```diff
@@ -92,6 +93,7 @@ public void markCanBeCompleted(long id) {
     Long start = starTimes.remove(opId);
     if (start != null) {
       stats.markRpcComplete(clock.nanoTime() - start);
```

Comment:
Is it safe to remove the DropWizard metrics?

Reply:
I think we need them, as they provide broader lifetime stats using samplings, opposed to the bounded recent time-window metrics we get using DescriptiveStatistics.